### DISCUSSION TITLE
Monitor and download partial incremental season packs

### DISCUSSION THREAD

**Discussion by daromanyuk** [Flexget/Flexget#3722] [Q&A]
Hi all!

I'm considering to move from Sonarr to flexget (sonarr is not so flexible ...)
I would like to clarify whether it's possible with flexget to monitor and download "partial incremental season packs". 
What I mean by this: 
in my region most torrent trackers build on top of  forum platforms and series there publishes as season packs right from the first episode..   For example "The last of us"  - in the tracker It available as single torrent file with title "The Last of Us (S1E1-3) (2023) WEBRip 1080p Ukr/Eng (S1E1-3)"   that contains 3 episodes.  When  next episode available the title is changed to  "The Last of Us (S1E1-**4**) (2023) WEBRip 1080p Ukr/Eng (S1E1-**4**)" etc..    When season finalized it updated to "The Last of Us (**S1**) (2023) WEBRip 1080p Ukr/Eng (**S1**)"

So my question - Is it possible to configure flexget to monitor updates in season pack and download new episodes? 

Thank you!

**Comment by daromanyuk** [comment]
I was digging through documentation and found that there is possible to set custom season-and-episodes parsing rules, that looks promising.  
have anyone done the same scenario as I have before ?

**Comment by daromanyuk** [comment]
Found this draft article https://www.flexget.com/Drafts/Season-pack-support
looks like that what i need. 

1. Do you know when this features will be available? 
2. what it the workaround to achieve what i asker above ?

**Reply by gazpachoking** [reply]
Your original link is the current season pack support we have implemented. https://www.flexget.com/Plugins/series/season_packs
Incremental season packs are not something that has any current built in support.
If you can figure out the how the rules you want should work, you may be able to tell flexget how to implement them though, it'll just be more verbose and require a bunch more reading, trial and error.
